File: src/interoperability/Main2.java
package interoperability;

public class Main2 {

    public static void main(String args[]) {
        System.out.println("interoperability");
        int sum = MainKt.add(2, 3);
        System.out.println("the sum is " + sum);
    }

    static int add(int a, int b) {
        return a + b;
    }

     static int area(int l, int b) {
        return l * b;
    }
}

File: src/oct29/Main2.kt
package oct29

fun main(args: Array<String>) {
    println("Learning function sin kotlin")
    findArea(2, 3)
    println(add(2, 3))
}

fun findArea(length: Int, breadth: Int) {
    println("The area is ${length * breadth}")
}

fun add(a: Int, b: Int): Int {
    println("The sum is ${a + b}")
    return a + b;
}